### PR TITLE
Finish receipts module

### PR DESCRIPTION
**Pull request by EvsenevDev**

### CODE REVIEW

File: SmartReceipts/Eureka Custom/ExchangeRateCell.swift [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    
+    weak var alertPresenter: QuickAlertPresenter?
+    
+    private let bag = DisposeBag()
+    private var error: CurrencyExchangeError?
+    
+    public override func setup() {
+        textField = valueField
+        titleLabel = label
+        titleLabel?.font = textLabel?.font
```
Comment (wbaumann) [inline SmartReceipts/Eureka Custom/ExchangeRateCell.swift]: Why set the font both here and two lines below?

File: SmartReceipts/Modules/Edit Receipt/EditReceiptView.swift [hunk 2]
```diff
@@ -56,6 +58,18 @@ final class EditReceiptView: UserInterface {
         }).addDisposableTo(disposeBag)
     }
     
+    private func configureTitle() {
+        var id: UInt!
+        if displayData.receipt == nil {
+            id = Database.sharedInstance().nextReceiptID()
```
Comment (wbaumann) [inline SmartReceipts/Modules/Edit Receipt/EditReceiptView.swift]: We may need to move this off the UI thread

File: SmartReceipts/Utils/WBAutocompleteHelper.m [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        _suggestionView.delegate = self;
+        return _suggestionView;
+    }
+}
+
+#pragma mark - Public methods
+
+-(void)textFieldDidBeginEditing:(UITextField *)textField {
+    if (textField == _field) {
+        // not needed anymore
```
Comment (wbaumann) [inline SmartReceipts/Utils/WBAutocompleteHelper.m]: Can we delete these comment fields?

File: SmartReceipts/Eureka Custom/ExchangeRateCell.swift (hunk 1 above)
Comment (EvsenevDev) [inline SmartReceipts/Eureka Custom/ExchangeRateCell.swift]: Oh, it's my mistake, sorry

File: SmartReceipts/Modules/Edit Receipt/EditReceiptView.swift (hunk 2 above)
Comment (EvsenevDev) [inline SmartReceipts/Modules/Edit Receipt/EditReceiptView.swift]: ok

File: SmartReceipts/Utils/WBAutocompleteHelper.m (hunk 3 above)
Comment (EvsenevDev) [inline SmartReceipts/Utils/WBAutocompleteHelper.m]: ok

**Pull request merged**